bin/internal/update_dart_sdk.ps1

```diff
@@ -20,17 +20,31 @@ $cachePath = "$flutterRoot\bin\cache"
 $dartSdkPath = "$cachePath\dart-sdk"
 $dartSdkLicense = "$cachePath\LICENSE.dart_sdk_archive.md"
 $engineStamp = "$cachePath\engine-dart-sdk.stamp"
-$engineRealm = (Get-Content "$flutterRoot\bin\internal\engine.realm")
 
-# Calculate the engine hash from tracked git files.
-Push-Location "$flutterRoot"
-if ($null -eq $Env:LUCI_CONTEXT) {
-    $engineVersion = (git merge-base HEAD master)
+# Test for fusion repository
+if ((Test-Path "$flutterRoot\DEPS" -PathType Leaf) -and (Test-Path "$flutterRoot\engine\src\.gn" -PathType Leaf)) {
+  # Calculate the engine hash from tracked git files.
+    Push-Location "$flutterRoot"
+    if ($null -eq $Env:LUCI_CONTEXT) {
+        $engineVersion = (git merge-base HEAD master)
```

Comment:
i thought we were gonna use `upstream/master`?

Reply:
Yeah, still havent' gotten there yet.